portfolio/src/main/webapp/styles/style.css

```diff
@@ -1,9 +1,145 @@
-#content {
```

Comment:
Could you sort the CSS selectors either by type (i.e. foo, .bar, #baz) or by relevant group? The current ordering is not very clear to me, and makes it harder to mentally parse.

Reply:
I will paste my most recent commit in response to this comment below. Please let me know if that makes sense to you or if you would like to discuss further. Thanks!

Commit 529ac25
> **Change order of CSS selectors for general elements and add comments for top navigation bar**
> 
> These changes were made to ensure readability with respect to the order of the CSS selectors. For general elements, the selectors were sorted by type from most general selector to most specific (element tag, class, id). For the rest of the CSS selectors, they were organized based on the associated HTML heirarchy. For example -- Top Navigation Bar: The first selector is the overall navigation bar container that has the id of 'outer-nav', the next two selectors are for the home logo and the right section of the nav-bar, as these are the child elements of the overall nav-bar. This recursive nature continues and is shared by the sidebar CSS selectors.